ebu_tt_live/bindings/__init__.py

```diff
@@ -579,6 +579,33 @@ class LiveStyledElementMixin(StyledElementMixin):
 # NOTE: Some of the code below includes handling of SMPTE time base, which was
 # removed from version 1.0 of the specification.
 
+class RoleMixin(object):
+    _computed_roles = set()
+
+    def _semantic_compute_roles(self, dataset):
+        current_roles = set()
+        if self.role is not None:
```

Comment:
Very minor point: for consistency in this change request, I think this can be `if self.role:` because `bool(None)` evaluates to `False` in Python these days, I learned today!

Reply:
you are right, updating